scripts/install-scenario

```diff
@@ -48,7 +48,7 @@ echo
 set -ex
 
 composer -n validate --working-dir=$dir --no-check-all --ansi
-composer -n --working-dir=$dir ${DEPENDENCIES} --prefer-dist --no-scripts
```

Comment:
Here, `${DEPENDENCIES}` might be `update`, so we should keep `--no-scripts` here.

I'm not sure how to solve the problem here. On an update, we'd want to run an update first with --no-scripts, followed by an install without --no-scripts.  For an install, we could just run without --no-scripts, I think.